package.json

```diff
@@ -12,12 +12,11 @@
     "karma": "karma",
     "karma.start": "karma start",
     "lint": "gulp tslint",
-    "postinstall": "tsd reinstall --clean && tsd link && tsd rebundle && gulp postinstall",
     "start": "gulp serve --env dev",
     "serve.dev": "gulp serve --env dev",
     "tasks.list": "gulp --tasks-simple",
     "test": "gulp test",
-    "tsd": "tsd"
+    "tsd": "gulp tsd"
```

Comment:
@ludohenin OK, I understand. Beware, though that `gulp tsd` is a proxy for the previous `tsd reinstall --clean && tsd link && tsd rebundle`. 

Should I create another task (for easy discovery)? If so, what name? 

Suggestion: `npm run tsd:reinstall` or `npm run tsd:update`